src/main/java/botto/command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,61 @@
+package botto.command;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import botto.BottoException;
+import botto.task.Deadline;
+import botto.task.Task;
+import botto.util.Dialog;
+import botto.util.Storage;
+import botto.util.TaskList;
+
+/**
+ * Command for adding a deadline
+ */
+public class AddDeadlineCommand implements Command {
+    private String command;
```

Comment:
Instead of passing a command String to each command object, you can consider passing a Task object, and leave the parsing logic below to your Parser class! That way, you can also merge AddDeadlineCommand, AddTodoCommand and AddEventCommand into a single AddCommand.